pages/sentinel_create_samples.vue

```diff
@@ -95,17 +95,20 @@ export default {
       const errored = resp.filter((obj) => Object.keys(obj).includes('errors'))
       if (errored.length > 0) {
         const msg = errored.map((e) => e.errors.join(', ')).join(', ')
-        this.alertMessage = msg
-        this.showDismissibleAlert = true
+        this.showAlert(msg, 'danger')
       }
 
       const successful = resp.filter((obj) => Object.keys(obj).includes('data'))
       if (successful.length > 0) {
         this.items = successful.map((obj) => obj.data).map((obj) => obj.data)
+        if (!errored.length) { this.showAlert('Sentinel samples successfully created in sequencescape', 'success') }
```

Comment:
Did we want to add an warning if it is a partical success saying the items in the table have been sucessfully added to SS, and these are the barcodes that have failed?